srml/contract/src/lib.rs

```diff
@@ -16,14 +16,14 @@
 
 //! # Contract Module
 //!
-//! The contract module provides functionality for the runtime to deploy and execute WebAssembly smart-contracts.
-//! The supported dispatchable functions are documented as part of the [`Call`](./enum.Call.html) enum.
```

Comment:
Why did you remove the `Call` enum here?

Reply:
I moved it to the Dispatchable Functions section so that you see a link from the brief summaries to the more detailed documentation.